src/sentry/api/serializers/models/organization_member/base.py

```diff
@@ -27,19 +28,24 @@ def get_attrs(
 
         # Preload to avoid fetching each user individually
         prefetch_related_objects(item_list, "user", "inviter")
-        users_set = {
-            organization_member.user
-            for organization_member in item_list
-            if organization_member.user_id
+        users_set = sorted(
```

Comment:
Why does the sorting matter if the value is a set?

Reply:
Primarily because the existing tests, unfortunately, seemed to be flaky and depend on ordering.  It's confusing to me since the previous implementation was also a set, but it was a set of `User` and not `int`, and for reasons that I suspect have something to do with `hash` implementations, it didn't work in the tests until I performed the sort resulting in a specific ordering for the tests.  I could change that in the tests, of course, bu the other reason I changed was also because I needed the set to become a `List` for typing reasons and... I was lazy about swapping i for `Iterable`? :PP  Like yeah, basically I made a call "no harm, strictly more productive" and moved forward.  If you think I should address the issue more fundamentally I can give i a shot.

Reply:
Nah I don't think the values being sorted is that important. It was strange to see `sorted()` on a data structure that doesn't have ordering.